resources/assets/coffee/react/_components/beatmapset-mapping.coffee

```diff
@@ -20,7 +20,7 @@ el = React.createElement
 
 bn = 'beatmapset-mapping'
 
-@BeatmapsetMapping = ({user, beatmapset}) ->
+@BeatmapsetMapping = ({user, beatmapset, beatmap}) ->
```

Comment:
oh, sorry I misunderstood what you said. that seems a bit strange to use the set's approved status (since it might differ from the actual map), but I can change it back if it's intentional